app/boards/arm/bt60/bt60_v1.dts

```diff
@@ -66,19 +67,6 @@
         >;
     };
 
-    split_transform: keymap_transform_4 {
```

Comment:
Is this layout not actually supported by this PCB? Was it an oversight in adding it originally?

Reply:
It's a minor variation of the all 1U layout and can be configured as an all 1u layout. whilst being a technically possible layout option I figured it was better to streamline it and just drop it, doing a discrete physical layout for every possible layout isn't really practical. PW never offered an explicit transform for the BT60V2 etc for this layout and it's not been an issue. The all 1u layout covers everything

Reply:
Just searched for it https://github.com/search?q=SPLIT_BKSP_RSHFT&type=code and I don't think anyone is actually using that layout